cluster/log-dump/log-dump.sh

```diff
@@ -0,0 +1,344 @@
+#!/bin/bash
```

Comment:
This is a move with some changes, right? Are you sure you deleted old 'log-dump.sh', or are you leaving it there for a reason?